deploy/kustomize/driver/pmem-csi.yaml

```diff
@@ -4,6 +4,26 @@ metadata:
   name: pmem-csi-intel-com-controller
   namespace: default
 ---
+# This role binding corresponds to what the following command creates:
```

Comment:
Isn't it good to keep these kinds of changes in a kustomize layer specific to openshift?

Reply:
That would imply that we maintain different deployments and deployment instructions for upstream Kubernetes and OpenShift, with the resulting user errors when they pick the wrong one.

I'm trying to avoid that.